404.html

```diff
@@ -3,12 +3,15 @@
 title: "Error 404"
 ---
 <div class="gray-wrapper">
-    <div class="container spaced">
-        <div class="row">
-            <div class="col">
-                <h2>Page not found :(</h2>
-                <p>The requested page could not be found.</p>
-            </div>
-        </div>
-    </div>
+	<div class="container spaced">
+		<div class="row">
+			<div class="col">
+				<h2>Page not found :(</h2>
+				<p>The requested page could not be found.</p>
+			</div>
+			<div>
+				<iframe class="embed-responsive-item" src="https://www.youtube.com/embed/t3otBjVZzT0?autoplay=1" frameborder="0" allowfullscreen></iframe>
+			</div>
+		</div>
+	</div>
 </div>
```

Comment:
It looks like you switched from spaces to tabs. Was that intentional?

Reply:
Yeah

Reply:
It looks like we don't use the tab charachter anywhere else in the repo, would you mind converting back to spaces to keep it consistent?
```
8 20:24 bionicpi@erebus:CSHPublicSite $ grep -rP '\t' .  
9 20:25 bionicpi@erebus:CSHPublicSite $ 
```

Reply:
> It looks like we don't use the tab character anywhere else in the repo, would you mind converting back to spaces to keep it consistent?
> 
> ```
> 8 20:24 bionicpi@erebus:CSHPublicSite $ grep -rP '\t' .  
> 9 20:25 bionicpi@erebus:CSHPublicSite $ 
> ```

Done!